hack/ci/windows.ps1

```diff
@@ -864,10 +903,12 @@ Try {
                 if (!($env:INTEGRATION_TESTFLAGS)) {
                     $env:INTEGRATION_TESTFLAGS = "-test.v"
                 }
-                Set-Location "$env:SOURCES_DRIVE`:\$env:SOURCES_SUBDIR\src\github.com\docker\docker"
                 $start=(Get-Date); Invoke-Expression ".\hack\make.ps1 -TestIntegration"; $Duration=New-Timespan -Start $start -End (Get-Date)
+                $IntTestsRunResult = $LastExitCode
                 $ErrorActionPreference = "Stop"
-                if (-not($LastExitCode -eq 0)) {
+                # Copy all the test results to TEMPORIG for archival
+                Copy-Item -Path "$env:SOURCES_DRIVE`:\$env:SOURCES_SUBDIR\src\github.com\docker\docker\bundles\junit-report*xml" -Destination $TEMPORIG
+                if (-not($IntTestsRunResult -eq 0)) {
```

Comment:
I realise this was already there, just curious; is there any difference between this, and

```powershell
if ($IntTestsRunResult -ne 0) {
```

Reply:
This is something we need to revisit immediately after these changes go through (for the issue where our Unix output does not match the actual results of test runs [passed or failed]). So if you feel strongly about using a particular pattern, please document it and we can use it for that change.

Reply:
Yeah, not a big deal for now, but `-ne 0` is easier to grasp than the "double negative" used currently.

Reply:
Absolutely - will make sure to change it with that fix.